submissions/crealime/html-css-popup/css/main.css

```diff
@@ -0,0 +1,290 @@
+  object-fit: cover;
+  border-radius: 50%
+}
+
+.top-menu__label {
+  display: block;
+  cursor: pointer
+}
+
+.top-menu__input {
+  position: absolute;
+  appearance: none
+}
+
+.top-menu__input:focus ~ .top-menu__label .top-menu__icon-box {
+  border: solid 1px #b4b4b4
+}
+
+.top-menu__input:checked ~ .products {
+  visibility: visible;
+  opacity: 1;
+  pointer-events: auto
+}
+
+.top-menu__icon-box {
+  display: block;
+  border: solid 1px rgba(52, 52, 52, 0);
+  border-radius: 0.25rem;
+  padding: 0.4375rem 0.625rem
+}
+
+.top-menu__icon {
+  width: 1.5rem;
+  height: 1.5rem;
+  object-fit: cover
+}
+
+.products {
+  position: absolute;
+  top: 4.375rem;
+  right: 0;
+  visibility: hidden;
+  pointer-events: none;
+  opacity: 0;
+  transition-duration: 300ms;
+  z-index: 2
+}
+
+@media only screen and (max-width: 558px) {
+  .products {
+    transform: translateX(calc(50% - 35px))
+  }
+}
+
+.products__ul {
+  background: #fff;
+  padding: 1.25rem;
+  display: grid;
+  min-width: 25rem;
+  grid-template-columns:33.33% 33.33% 33.33%
+}
+
+@media only screen and (max-width: 558px) {
+  .products__ul {
+    min-width: 20rem
+  }
+}
+
+.products__ul:before {
+  content: "";
+  position: absolute;
+  top: -1.75rem;
+  right: 0.4375rem;
+  border: 15px solid rgba(0, 0, 0, 0);
+  border-bottom: 15px solid #fff;
+  z-index: -1
+}
+
+@media only screen and (max-width: 558px) {
+  .products__ul:before {
+    right: 0;
+    transform: translateX(-7.9375rem)
+  }
+}
+
+.products__li {
+  list-style-type: none
+}
+
+.products__li:nth-child(n+10) {
+  display: none
+}
+
+.products__link {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.625rem;
+  border: solid 1px rgba(52, 52, 52, 0);
+  transition-duration: 300ms
+}
+
+.products__link:focus {
+  border: solid 1px #cdcdcd
+}
+
+.products__icon {
+  max-width: 100%;
+  max-height: 4.375rem;
+  margin-bottom: 0.625rem
+}
+
+.products__input {
+  position: absolute;
+  appearance: none
+}
+
+.products__input:checked ~ .products__ul .products__li:nth-child(n+10) {
+  display: block
+}
+
+.products__input:checked ~ .products__ul {
+  min-width: 26.0625rem;
+  height: 25rem;
+  overflow-y: auto
+}
+
+@media only screen and (max-width: 558px) {
+  .products__input:checked ~ .products__ul {
+    height: 24.25rem;
+    width: 20rem;
+    min-width: 20rem
+  }
+}
+
+.products__input:checked ~ .products__btn .products__box:after {
+  content: "Less"
+}
+
+.products__input:focus ~ .products__btn .products__box {
+  background: #b4b4b4
+}
+
+.products__box {
+  display: block;
+  position: relative;
+  width: 100%;
+  height: 2.5rem;
+  background: #cdcdcd;
+  cursor: pointer;
+  transition-duration: 300ms
+}
+
+.products__box:hover, .products__box:focus {
+  background: #b4b4b4
+}
+
+.products__box:after {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  content: "More";
+  transform: translateY(-50%) translateX(-50%)
```

Comment:
what is this for?

Reply:
This is for positioning the pseudo-element :after with text "More" centered on the parent.